wca-regulations.md

```diff
@@ -221,7 +221,9 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 
 ## <article-4><scrambling><scrambling> Article 4: Scrambling
 
-- 4a) A scrambler applies scramble sequences to the puzzles.
+- 4a) A scrambler applies scramble sequences to the solved puzzles.
+    - 4a1) Exception: for extra attempts for 5x5x5 Cube, 6x6x6 Cube, 7x7x7 Cube and Megaminx, the scramble sequence may be applied to an unsolved puzzle, at the discretion of the WCA Delegate. This exception does not apply to scramble sequences which are not used for extra attempts.
```

Comment:
The part _This exception does not apply to scramble sequences which are not used for extra attempts._ is redundant since the first sentence already says _for extra attempts_
Additionally, the **double negative** is super confusing. If it needs to be kept, simplify it to: _This exception only applies to scramble sequences for extra attempts_ (or similar)